crates/jstzd/src/protocol/mod.rs

```diff
@@ -142,25 +160,76 @@ impl ProtocolParameterBuilder {
         };
         Ok(raw_json)
     }
+
+    fn inject_bootstrap_accounts(
```

Comment:
Prefer `append` or `write` over `inject` as inject can have the meaning of injecting an operation.

Reply:
Changed to `merge`.